common/rya.api/src/main/java/org/apache/rya/api/RdfCloudTripleStoreConfiguration.java

```diff
@@ -376,6 +377,15 @@ public void setInferSubPropertyOf(final Boolean val) {
         setBoolean(INFER_INCLUDE_SUBPROPOF, val);
     }
 
+    public Boolean hasSelf() {
```

Comment:
can you change this to isInferHasSelf() so its more in line with other flags for inference?